sway/desktop/xdg_shell.c

```diff
@@ -148,7 +148,15 @@ static uint32_t configure(struct sway_view *view, double lx, double ly,
 	if (xdg_shell_view == NULL) {
 		return 0;
```

Comment:
@emersion If 0 does not mean an unsuccessful configure, then this code here is wrong. Should we give 0 a special meaning and document that or should we add a new `bool check_valid_configure()` function to check invalid conditions?